modules/nodes-pods-autoscaling-about.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id='nodes-pods-autoscaling-about_{context}']
+= Understanding horizontal pod autoscalers
+
+You can create a horizontal pod autoscaler to specify the minimum and maximum number of pods you want to run, as well as the
+CPU utilization or memory utilization your pods should target.
+
+HPA, automatically scale the number of pods up and down for a specific Replication Controller or DeploymentConfig. You can specify the minimum and maximum number of pods 
```

Comment:
I think we should probably delete this paragraph. It restates the info from the preceding paragraph and starts off awkwardly. Or combine them if you want to keep the part about it updating the rc or dc with the replica value